components/scheduler/recurrence.md

```diff
@@ -0,0 +1,432 @@
+---
+title: Recurrence
+page_title: Scheduler Recurrence
+description: Overview of the Scheduler for Blazor.
```

Comment:
This is probably a leftover from the template. Aim for around 100-150 characters.